Dockerfile.rpms

```diff
@@ -4,6 +4,7 @@ FROM registry.ci.openshift.org/origin/4.12:machine-config-operator as mcd
 FROM registry.access.redhat.com/ubi8/ubi-minimal
 WORKDIR /rpms
 COPY crio.repo /etc/yum.repos.d
-RUN microdnf download crio cri-tools --archlist=x86_64
+RUN microdnf download crio cri-tools --archlist=$(uname -m) \
```

Comment:
Maybe we should be fetching all arches, add another 'ARG arch x86_64` and copy the necessary RPM in `Dockerfile.template` so that:
* `okd-rpms` would have all necessary arches
* `machine-os-content` could be rebuilt for other arches?

Reply:
I think once we have multi-arch payloads, there will be manifestlisted multi-arch images for the okd-rpms image too, so each image only needs to carry the RPMs specific to its arch.

Reply:
Yes, Lorbus' way is the current approach used elsewhere. I'd say it's better, in general, not to bring other-arches "assets" into an amd64 image even though, in this case, it could work fine.